### PR TITLE
Update Verrazzano acceptance test to use Kind with metallb

### PR DESCRIPTION
**Pull request by guoyongzhang**
# Description

In Verrazzano build pipeline, update acceptance test to use metallb to replace nodeport for nginx and istio ingress access in KinD.

Fixes VZ-2146

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [x] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [x] Addressed the requirement and meets the acceptance criteria
- [x] Does not introduce unrelated or spurious changes
- [x] Does not introduce any unapproved dependency
- [x] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/config/scripts/install-metallb.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```
Comment (markxnelson) [inline tests/e2e/config/scripts/install-metallb.sh:3]: should not have 2020

File: tests/e2e/config/scripts/install-metallb.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+set -e
+
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.9.5/manifests/namespace.yaml
```
Comment (markxnelson) [inline tests/e2e/config/scripts/install-metallb.sh:9]: can you confirm that we only use this in tests, and not anywhere else, please?

File: tests/e2e/pkg/api.go [hunk 3]
```diff
@@ -25,7 +28,9 @@ type ApiEndpoint struct {
 
 // GetApiEndpoint returns the ApiEndpoint stub with AccessToken
 func GetApiEndpoint() *ApiEndpoint {
-	keycloakURL := fmt.Sprintf("https://keycloak.%s.%s/auth/realms/%s/protocol/openid-connect/token", EnvName, DnsZone, realm)
+	ingress, _ := GetKubernetesClientset().ExtensionsV1beta1().Ingresses("keycloak").Get(context.TODO(), "keycloak", v1.GetOptions{})
```
Comment (markxnelson) [inline tests/e2e/pkg/api.go:31]: is there any significance to using "keycloak" here - or did you just need any of the endpoints in order to get the ip address?

File: tests/e2e/config/scripts/install-metallb.sh (hunk 1 above)
Comment (guoyongzhang) [reply tests/e2e/config/scripts/install-metallb.sh]: fixed

File: tests/e2e/config/scripts/install-metallb.sh (hunk 2 above)
Comment (guoyongzhang) [reply tests/e2e/config/scripts/install-metallb.sh:9]: yes, the test is the only place to use it.

File: tests/e2e/pkg/api.go (hunk 3 above)
Comment (guoyongzhang) [reply tests/e2e/pkg/api.go:31]: accessing keycloak to get access_token that is needed by api endpoint.

**Review by markxnelson — APPROVED** [review]

**Pull request merged**